test/config.yaml

```diff
@@ -0,0 +1,86 @@
+#### General Settings ####
+# location of system level settings
+settings:
+  title: My Very Cool Project
+  Author: Bob
+
+  # path like settings
+  data: /data/bob/original_data
+
+  # conda environment names
+  python2: py2.7
+
+  # names to access specific envs by
+  env: HOME
```

Comment:
For sure I would leave this out initially, I was trying to think of all possible settings. May come across some misc perl program or something that needs these envs, but would also be at the rule level.